### PR TITLE
Refactor params reading in claims hosp

### PR DESCRIPTION
**Pull request by chinandrew**
### Description
Refactor claims hosp so that `run_module()` accepts a `params` argument corresponding to parameters read from the json file. 

### Changelog
- Change signature of `run_module()`
- Document required and optional parameters in comments of `run_module()`
- Remove use of `params.json` in tests now that `run_module()` does not read params

### Fixes
-  Partially addresses #814.

### CODE REVIEW

File: claims_hosp/delphi_claims_hosp/run.py [hunk 1]
```diff
@@ -11,17 +11,42 @@
...
+               If set to null, end date is derived from drop date and n_waiting_days.
+            - "drop_date": str or null, YYYY-MM-DD format, date data is dropped. If set to
+               null, current day minus 40 hours is used.
+            - "n_backfill_days": int, number of past days to generate estimates for.
+            - "n_waiting_days": int, number of most recent days to skip estimates for.
+            - "write_se": bool, whether to write out standard errors.
+            - "obfuscated_prefix": str, prefix for signal name if write_se is True.
+            - "parallel": bool, whether to update sensor in parallel.
+            - "geos": list of str, geographies to generate sensor for.
+            - "weekday": list of bool, whether to adjust for weekday effects.
```
Comment (sgsmob) [inline claims_hosp/delphi_claims_hosp/run.py:47]: Why is this list of bools rather than just a bool?

Comment (chinandrew) [reply claims_hosp/delphi_claims_hosp/run.py:47]: don't know original design motivation, but the weekday value is [True, False] and then there's a line which is ` for weekday in params["indicator"]["weekday"]:` and loops through a version with and a version without weekday corrections. 

Could probably make the docstring more clear, but now sure what the best wording would be

Comment (sgsmob) [reply claims_hosp/delphi_claims_hosp/run.py:47]: Maybe "list of bool, which weekday adjustments to perform.  For each value in the list, signals will be generated with weekday adjustments (`True`) or without adjustments (`False`).

Comment (chinandrew) [reply claims_hosp/delphi_claims_hosp/run.py]: :+1: updated, thanks

File: claims_hosp/tests/test_update_indicator.py [hunk 2]
```diff
@@ -16,7 +16,14 @@
 
 CONFIG = Config()
 CONSTANTS = GeoConstants()
-PARAMS = read_params()
+PARAMS = {
+    "indicator": {
+        "input_file": "test_data/SYNEDI_AGG_INPATIENT_11062020_1451CDT.csv.gz",
+        "drop_date": "2020-06-11",
+        "obfuscated_prefix": "foo_obfuscated"
+
```
Comment (sgsmob) [inline claims_hosp/tests/test_update_indicator.py:24]: drop newline here

**Review by sgsmob — APPROVED** [review]
just one nit, lgtm

File: claims_hosp/tests/test_update_indicator.py (hunk 2 above)
Comment (chinandrew) [reply claims_hosp/tests/test_update_indicator.py:24]: removed

**Pull request merged**